### PR TITLE
Update DG to include filter and link command

### PR DESCRIPTION
**Pull request by xyT-T**

### CODE REVIEW

File: docs/DeveloperGuide.md
Comment (rionshocker) [inline docs/DeveloperGuide.md:1]: Missing the user stories for filter and link commands

**Review by rionshocker — CHANGES_REQUESTED** [review]
Apart from the missing user stories, the rest LGTM!

File: docs/DeveloperGuide.md
Comment (xyT-T) [reply docs/DeveloperGuide.md:1]: ok! user stories added.

**Review by rionshocker — APPROVED** [review]
LGTM!

**Pull request merged**